### PR TITLE
Add TGIS to openshift-ci

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Draft for including TGIS in the openshift-ci build

### CODE REVIEW

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-text-generation-inference-main-images

**Comment by vaibhavjainwiz** [comment]
/pj-rehearse pull-ci-opendatahub-io-text-generation-inference-main-unit

File: ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      channel: fast
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      cpu: "20"
+      memory: 16Gi
+    requests:
+      cpu: "20"
+      memory: 16Gi
```
Comment (Jooho) [inline ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml:34]: After you build the tgis image, you should send it to opendatahub quay.io
in order to send the built image, please refer this https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml#L78-L95

with this, we can see pr tag/fast tag

File: ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    namespace: openshift
+    tag: golang-1.13
+images:
+- build_args:
+  - name: DOCKER_BUILDKIT
+    value: "1"
+  - name: BUILDKIT_PROGRESS
+    value: plain
+  - name: PROTOC_VERSION
+    value: "23.2"
```
[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml]: done

File: ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml:32]: done

**Review by Jooho — APPROVED** [review]
/lgtm

**Pull request merged**